packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/autocomplete/autocomplete.css

```diff
@@ -20,11 +20,13 @@
 		&.ck-search__results_n {
 			border-bottom-left-radius: 0;
 			border-bottom-right-radius: 0;
+			margin-bottom: -1px;
```

Comment:
Why do we need it?

Reply:
When the results panel is displayed next to the input via `bottom: 100%;` or `top: 100%`, you get a double border: one pixel from the input, another from the panel. This mitigates the duplication. We don't have a custom prop for border-width, so I cannot use `calc( -1 * var(--some-custom-prop ) )` here.